error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/JUnitToAssertJRules.java

```diff
@@ -78,8 +62,13 @@ T before(String message) {
       return Assertions.fail(message);
     }
 
+    // XXX: Add `@UseImportPolicy(STATIC_IMPORT_ALWAYS)` once
+    // https://github.com/google/error-prone/pull/3584 is resolved. Until that time, statically
+    // importing AssertJ's `fail` is likely to clash with an existing static import of JUnit's
+    // `fail`. Note that combining Error Prone's `RemoveUnusedImports` and
+    // `UnnecessarilyFullyQualified` checks and our `StaticImport` check will anyway cause the
```

Comment:
I was worried about the order of checks and potential clashes still, but luckily the `StaticImport` check [guards](https://github.com/PicnicSupermarket/error-prone-support/blob/12f2feaacfc819cd8ae2733396f933a5c155f67f/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StaticImport.java#L250) against that.